pennylane/fermi/fermionic.py

```diff
@@ -303,6 +303,132 @@ def to_mat(self, n_orbitals=None):
 
         return mat
 
+    def commute(self, source, target):
+        r"""Apply anti-commuting relations until the operator in position source is in position target.
+
+        Args:
+            source (int): The position of the operator to be commuted.
```

Comment:
NIT: To me, it's a little strange that we write "position" in the docstring but then we use "indice" for the error messages, you know what I mean? I feel like we should be consistent; personally I would change the error messages to use position instead since we use position for the docs in FermiWord/FermiSentence as well iirc.